addons/crm/tests/common.py

```diff
@@ -219,6 +219,18 @@ def setUpClass(cls):
             'country_id': cls.env.ref('base.us').id,
             'zip': '97648',
         })
+        cls.contact_without_email = cls.env['res.partner'].create({
```

Comment:
Arguably, since it's just one test you could just remove the email from an existing partner in the test itself